kwikapi/protocols.py

```diff
@@ -35,6 +35,18 @@ def get_record_separator(self):
     def get_mime_type(self):
         pass
 
+    @staticmethod
+    def should_wrap():
+        '''
+        While returning the response the,
+        kwikapi will wrap the response as -
+        {success: value, result: value}
+
+        This is applicable to all the protocols,
+        except the raw protocol
```

Comment:
@shivam05011996 This line here about raw protocol is not necessary. Later if we add another proto that also doesn't need wrapping, this statement will no longer be true.

Reply:
I have modified the docstring, such that, it will remain generalized for future protocol inclusions.